client/package.json

```diff
@@ -39,5 +41,15 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "and": "0.0.3",
```

Comment:
Again you have some bad packages from invalid copy paste. You did the same mistake. Fix by deleting not needed packages